.github/workflows/publish-image.yml

```diff
@@ -0,0 +1,63 @@
+name: Publish Docker image
+
+on:
+  pull_request:
+    types:
+      - opened
+      - reopened
+      - ready_for_review
+      - synchronize
+  merge_group:
+  workflow_dispatch:
```

Comment:
I think this should be on push to main?

Reply:
Yep, this is just to trigger the actions during testing

Reply:
Yeah I realised after this that you undrafted it to test, and it's probably not actually ready for review - sorry!